composer.json

```diff
@@ -81,6 +81,7 @@
         ]
     },
     "suggest": {
-        "ext-pcntl": "Required to ensure that data is cleared when cancelling the build process."
+        "ext-pcntl": "Required to ensure that data is cleared when cancelling the build process.",
+        "illuminate/validation": "Require this package to be able to use validation rules directly in the ask method"
```

Comment:
The `illuminate/validation` package also needs to be added to the `require-dev` key. 👍 This should fix the Travis CI builds.